frontend/ui/widget/iconwidget.lua

```diff
@@ -35,7 +35,7 @@ local IconWidget = ImageWidget:extend{
     -- be overriden by callers.
     width = Screen:scaleBySize(DGENERIC_ICON_SIZE), -- our icons are square
     height = Screen:scaleBySize(DGENERIC_ICON_SIZE),
-    alpha = true, -- our icons have a transparent background
+    alpha = false, --- @note: our icons have a transparent background, but we flatten them at caching time, and this flag is only checked at blitting time.
```

Comment:
No issue with transparent PNG icons (themes, users icons)
(I guess you could imply alpha=true or do_alpha from is_icon then, dunno which is cleaner.)

Reply:
Yeah, that was my first implementation, but since nothing else really sets is_icon, I went straight to the source ;).

The flattening would happen for PNGs, too, so, not an issue either on that front, behavior is consistent.